image/templates/helm/shared/templates/02-scanner-v4-06-db-pvc.yaml

```diff
@@ -24,7 +24,6 @@ metadata:
     {{- include "srox.labels" (list . "persistentvolume" $pvName) | nindent 4 }}
   annotations:
     {{- include "srox.annotations" (list . "persistentvolume" $pvName) | nindent 4 }}
-    "helm.sh/hook": "pre-install,pre-upgrade"
     "helm.sh/resource-policy": keep
     "helm.sh/hook-delete-policy": never
```

Comment:
I think this should be removed, when the resource isn't a hook any longer, right?

Reply:
Removed it.